packages/go/index.ts

```diff
@@ -524,7 +538,7 @@ export async function startDevServer(
   }
   const analyzedRaw = await getAnalyzedEntrypoint(
     workPath,
-    entrypointWithExt,
+    (files[entrypointWithExt] as FileFsRef).fsPath,
```

Comment:
I think using `files` here is incorrect. Instead we should keep the call site the same and if we need the absolute path, the `getAnalyzedEntrypoint()` can do `join(workPath, entrypoint)` instead.

Reply:
@styfle  Hi, I used `files` to get the absolute path here only after reading the build function. 

The build function calls `getAnalyzedEntrypoint()` with the argument `filePath` which is the absolute path
>https://github.com/vercel/vercel/blob/main/packages/go/index.ts#L147

```
  analyzed = await getAnalyzedEntrypoint(
      workPath,
      downloadedFiles[entrypoint].fsPath,
      goModAbsPathDir
    );
```

Is it better to pass the absolute path when calling the function than to join inside the `getAnalyzedEntrypoint` function?

Reply:
I think it’s best to join inside the func since there are two callers so this will consolidate